VONK-2696: added SQL server guidelines to documentation (#20)
FirelyTeam/firely-docs-firely-server: performance/sqlserver/memory_settings.rst

Memory Settings
===============

> The total ammoun of memory needed by SQL Server depends on your use case and (expected) workload. SQL Server will claim memory from the OS when needed but will not give it back to the OS once claimed. When not specifying a limit you will run the risk of starving the host server and OS of enough free memory to operate efficently. It is therefore strongly advised to limit the maximum ammount SQL Server will use.

Reviewer: - Typo: 'ammoun' -> 'amount'
- Style: 'When not specifying a limit' -> 'If no limit is specified'
- Typo: 'efficently' -> 'efficiently'
- Typo: 'ammount' -> 'amount'
- Style: 'will use' -> 'is allowed to use'

Author: Corrected typo's

To set a fixed amount of memory using SSMS [#]_:

#. In Object Explorer, right-click a server and select Properties.
#. Click the Memory node.

> #. Under Server Memory Options, enter the amount that you want for Minimum server memory and Maximum server memory.

Reviewer: - Capitalization: 'Minimum' -> 'minimum'
- Capitalization: 'Maximum' -> 'maximum'

Author: It references the field names in the screenshot. I have put quotes around in it and fully matched the complete name which hopefully makes it a bit more clearer.

.. image:: ../../images/perf_sql_configure-memory-in-ssms.png
    :alt: Configure memory in SSMS

Use the default settings to allow SQL Server to change its memory requirements dynamically based on available system resources. It is recommended to set a max server memory as detailed above.

> As for the ammount of memory to reserve for the OS, Windows Server with Desktop Experience requires a minimum of 2 GB [#]_, but more is better so aim for at least 4 GB. For example if your server has 64GB of memory, set the amount of memory in SQL Server to be 60,000MB (i.e. 64GB - 4GB = 60GB)

Reviewer: ```suggestion
Windows Server with Desktop Experience requires a minimum memory size of 2 GB [#]_, but 4 GB or more is recommended (more is better). Example: If your server has 64GB of memory, set the amount of memory in SQL Server to 60,000MB (i.e. 64GB - 4GB = 60GB)
```
